cmd/gateway/manta/gateway-manta.go

```diff
@@ -205,23 +212,28 @@ func (g *Manta) NewGatewayLayer(creds auth.Credentials) (minio.ObjectLayer, erro
 			logger.LogIf(ctx, err)
 			return nil, err
 		}
-	}
-
-	tc, err := storage.NewClient(&triton.ClientConfig{
-		MantaURL:    endpoint,
-		AccountName: creds.AccessKey,
-		Signers:     []authentication.Signer{signer},
-	})
-	if err != nil {
-		return nil, err
-	}
+  }
+
+  config := &triton.ClientConfig{
+    MantaURL:     endpoint,
+    AccountName:  creds.AccessKey,
```

Comment:
Can you run `gofmt`  on the files that you changed? code looks off from regular indentation.